### PR TITLE
Added links leading to the Toloka portal pages to enhance SEO

### PR DESCRIPTION
**Pull request by alexandervnuchkov**
Added links leading to the Toloka portal pages to enhance SEO (TOLOKAMARKETING-1508)

## Checklist

- [x] I have read the **CONTRIBUTING** document
- [x] I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by dustalov** [comment]
Please don't do that.

File: crowdkit/aggregation/classification/gold_majority_vote.py [hunk 1]
```diff
@@ -13,7 +13,7 @@
 
 @attr.s
 class GoldMajorityVote(BaseClassificationAggregator):
-    r"""The **Gold Majority Vote** model is used when a golden dataset (ground truth) exists for some tasks.
+    r"""The **Gold Majority Vote** model is used when a golden [dataset](https://toloka.ai/datasets) (ground truth) exists for some tasks.
```
Comment (dustalov) [inline crowdkit/aggregation/classification/gold_majority_vote.py:16]: This link does not contribute to the documentation, please remove it. Below you have really good examples of links, but this one is too much.

File: crowdkit/aggregation/classification/m_msr.py [hunk 2]
```diff
@@ -34,8 +34,8 @@ class MMSR(BaseClassificationAggregator):
     And the aggregation is weighted majority voting with weights equal to
     $\log \frac{(M-1)s_i}{1-s_i}$.
 
-    Q. Ma and Alex Olshevsky. Adversarial Crowdsourcing Through Robust Rank-One Matrix Completion.
-    *34th Conference on Neural Information Processing Systems (NeurIPS 2020)*
+    Q. Ma and Alex Olshevsky. Adversarial [Crowdsourcing](https://toloka.ai/knowledgebase/crowdsourcing-concepts/) Through Robust Rank-One Matrix Completion.
+    *34th Conference on Neural Information Processing Systems ([NeurIPS](https://toloka.ai/blog/six-papers-from-neurips/) 2020)*
```
Comment (dustalov) [inline crowdkit/aggregation/classification/m_msr.py:38]: This link does not contribute to the documentation, please remove it.

File: crowdkit/aggregation/classification/mace.py [hunk 3]
```diff
@@ -66,7 +66,8 @@ def decode_distribution(gold_label_marginals: pd.DataFrame) -> pd.DataFrame:
 
 @attr.s
 class MACE(BaseClassificationAggregator):
-    r"""The **Multi-Annotator Competence Estimation** (MACE) model is a probabilistic model that associates each worker with a label probability distribution.
+    r"""The **Multi-[Annotator](https://toloka.ai/blog/what-does-a-data-annotator-do) Competence Estimation** (MACE) model
```
Comment (dustalov) [inline crowdkit/aggregation/classification/mace.py:69]: This link does not contribute to the documentation, please remove it.

File: crowdkit/aggregation/classification/zero_based_skill.py [hunk 4]
```diff
@@ -22,7 +22,7 @@ class ZeroBasedSkill(BaseClassificationAggregator):
 
     {% note info %}
 
-    It is necessary that all workers in the dataset that is sent to `predict` exist in responses to
+    It is necessary that all workers in the [dataset](https://toloka.ai/datasets) that is sent to `predict` exist in responses to
```
Comment (dustalov) [inline crowdkit/aggregation/classification/zero_based_skill.py:25]: This link does not contribute to the documentation, please remove it.

File: crowdkit/aggregation/image_segmentation/segmentation_rasa.py [hunk 5]
```diff
@@ -21,7 +21,7 @@ class SegmentationRASA(BaseImageSegmentationAggregator):
     for texts.
 
     Jiyi Li.
-    A Dataset of Crowdsourced Word Sequences: Collections and Answer Aggregation for Ground Truth Creation.
+    A [Dataset](https://toloka.ai/datasets) of Crowdsourced Word Sequences: Collections and Answer Aggregation for Ground Truth Creation.
```
Comment (dustalov) [inline crowdkit/aggregation/image_segmentation/segmentation_rasa.py:24]: This link does not contribute to the documentation, please remove it.

File: crowdkit/aggregation/pairwise/noisy_bt.py [hunk 6]
```diff
@@ -22,7 +22,7 @@ class NoisyBradleyTerry(BasePairwiseAggregator):
 
     Examples:
         The following example shows how to aggregate results of comparisons **grouped by some column**.
-        In the example the two questions `q1` and `q2` are used to group the labeled data.
+        In the example the two questions `q1` and `q2` are used to group the [labeled data](https://toloka.ai/data-labeling-platform).
```
Comment (dustalov) [inline crowdkit/aggregation/pairwise/noisy_bt.py:25]: This link does not contribute to the documentation, please remove it.

File: crowdkit/datasets/load_dataset.py [hunk 7]
```diff
@@ -11,7 +11,7 @@
 
 
 def load_dataset(dataset: str, data_dir: Optional[str] = None) -> Tuple[pd.DataFrame, pd.Series]:
-    """Downloads a dataset from remote and loads it into Pandas objects.
+    """Downloads a [dataset](https://toloka.ai/datasets) from remote and loads it into Pandas objects.
```
Comment (dustalov) [inline crowdkit/datasets/load_dataset.py:14]: This is a very well-placed link as `load_dataset()` downloads data from this place. Please keep it.

File: crowdkit/datasets/load_dataset.py [hunk 8]
```diff
@@ -31,6 +31,6 @@ def load_dataset(dataset: str, data_dir: Optional[str] = None) -> Tuple[pd.DataF
 
 
 def get_datasets_list() -> List[Tuple[str, str]]:
-    """Returns a list of available datasets in format [(name, description)]."""
+    """Returns a list of available [datasets](https://toloka.ai/datasets) in format [(name, description)]."""
```
Comment (dustalov) [inline crowdkit/datasets/load_dataset.py:34]: Same as above in `load_dataset()`, good job.

File: crowdkit/metrics/data/_classification.py [hunk 9]
```diff
@@ -209,7 +209,7 @@ def uncertainty(
 
 def alpha_krippendorff(answers: pd.DataFrame,
                        distance: Callable[[Hashable, Hashable], float] = binary_distance) -> float:
-    """Inter-annotator agreement coefficient (Krippendorff 1980).
+    """Inter-[annotator](https://toloka.ai/blog/what-does-a-data-annotator-do) agreement coefficient (Krippendorff 1980).
```
Comment (dustalov) [inline crowdkit/metrics/data/_classification.py:212]: This link does not contribute to the documentation, please remove it.

File: crowdkit/datasets/load_dataset.py (hunk 7 above)
Comment (pilot7747) [reply crowdkit/datasets/load_dataset.py:14]: No, please don't. We're planning to add datasets that are widely used by the community but weren't collected on Toloka. In this case, they won't be listed on https://toloka.ai/datasets and this change won't make sense.

File: crowdkit/datasets/load_dataset.py (hunk 8 above)
Comment (pilot7747) [reply crowdkit/datasets/load_dataset.py:34]: No, see above

**Comment by pilot7747** [comment]
As I mentioned in line-comments, even links in `load_dataset.py` don't contribute to the documentation. Overall, I would close this PR.

File: crowdkit/datasets/load_dataset.py (hunk 7 above)
Comment (dustalov) [reply crowdkit/datasets/load_dataset.py:14]: Agreed.

File: crowdkit/datasets/load_dataset.py (hunk 8 above)
Comment (dustalov) [reply crowdkit/datasets/load_dataset.py:34]: Agreed.

**Comment by alexandervnuchkov** [comment]
Oh, I see. Thank you for the review, I close the PR as inappropriate

**Pull request closed without merging**